### PR TITLE
Improvements to dataset tables

### PR DESCRIPTION
**Pull request by adamjstewart**
Various improvements to dataset tables in the docs. Will comment in-line.

### CODE REVIEW

File: docs/api/datasets.rst [hunk 1]
```diff
@@ -164,8 +164,8 @@ Kenya Crop Type
 
 .. autoclass:: CV4AKenyaCropType
 
-Deep Globe Land Cover Challenge
```
Comment (adamjstewart) [inline docs/api/datasets.rst:167]: Name was too long, cut off in miniviewer.

File: docs/api/geo_datasets.csv [hunk 2]
```diff
@@ -1,17 +1,18 @@
 Dataset,Type,Source,Size (px),Resolution (m)
-`Aboveground Woody Biomass`_,Mask,"Landsat, LiDAR","~40,000x40,000",~30
+`Aboveground Woody Biomass`_,Mask,"Landsat, LiDAR","40,000x40,000",30
 `Aster Global DEM`_,Mask,Aster,"3,601x3,601",30
-`Canadian Building Footprints`_,Labels,Generated,,
```
Comment (adamjstewart) [inline docs/api/geo_datasets.csv:4]: "Labels" is ambiguous. When I hear "labels" I think classification or regression. I think "geometries" is more clear for this dataset. Also, "generated" doesn't tell me anything.

File: docs/api/geo_datasets.csv [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
-`Chesapeake Land Cover`_,"Imagery, Labels",,,1
```
Comment (adamjstewart) [inline docs/api/geo_datasets.csv:5]: I think "mask" is better than "labels" for semantic segmentation problems

File: docs/api/geo_datasets.csv [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
-`Global Mangrove Distribution`_,Mask,Generated,,3
-`Cropland Data Layer`_,Labels,Aerial,,
```
Comment (adamjstewart) [inline docs/api/geo_datasets.csv:7]: Imagery is created using Landsat, so same resolution as Landsat

File: docs/api/geo_datasets.csv [hunk 5]
```diff
@@ -1,17 +1,18 @@
...
+`Chesapeake Land Cover`_,"Imagery, Mask",NAIP,-,1
+`Global Mangrove Distribution`_,Mask,"Remote Sensing, In Situ Measurements",-,3
+`Cropland Data Layer`_,Mask,Aerial,-,30
+`EDDMapS`_,Points,Citizen Scientists,-,-
+`EnviroAtlas`_,"Imagery, Mask","NAIP, NLCD, OpenStreetMap",-,1
+`Esri2020`_,Mask,Sentinel-2,-,10
+`EU-DEM`_,Mask,"Aster, SRTM, Russian Topomaps",-,25
+`GBIF`_,Points,Citizen Scientists,-,-
+`GlobBiomass`_,Mask,Landsat,"45,000x45,000",100
+`iNaturalist`_,Points,Citizen Scientists,-,-
```
Comment (adamjstewart) [inline docs/api/geo_datasets.csv:14]: iNaturalist was missing from the table.

For GBIF/EDDMapS/iNaturalist I used "points" for the dataset type and "citizen scientists" for the label sources

File: docs/api/non_geo_datasets.csv [hunk 6]
```diff
@@ -1,34 +1,34 @@
 Dataset,Task,Source,# Samples,# Classes,Size (px),Resolution (m),Bands
 `ADVANCE`_,C,"Google Earth, Freesound","5,075",13,512x512,0.5,RGB
-`Benin Cashew Plantations`_,S,Airbus Pléiades,70,6,"1,186x1,122",0.5,MSI
+`Benin Cashew Plantations`_,S,Airbus Pléiades,70,6,"1,122x1,186",10,MSI
```
Comment (adamjstewart) [inline docs/api/non_geo_datasets.csv:3]: If you run `gdalinfo` on the files in @calebrob6's VM, the pixel size is 10, not 0.5. Also, dimensions are reversed.

File: docs/api/non_geo_datasets.csv [hunk 7]
```diff
@@ -1,34 +1,34 @@
...
+`Deep Globe Land Cover`_,S,DigitalGlobe +Vivid,803,7,"2,448x2,448",0.5,RGB
+`DFC2022`_,S,Aerial,"3,981",15,"2,000x2,000",0.5,RGB
 `ETCI2021 Flood Detection`_,S,Sentinel-1,"66,810",2,256x256,5--20,SAR
 `EuroSAT`_,C,Sentinel-2,"27,000",10,64x64,10,MSI
 `FAIR1M`_,OD,Gaofen/Google Earth,"15,000",37,"1,024x1,024",0.3--0.8,RGB
 `Forest Damage`_,OD,Drone imagery,"1,543",4,"1,500x1,500",,RGB
 `GID-15`_,S,Gaofen-2,150,15,"6,800x7,200",3,RGB
 `IDTReeS`_,"OD,C",Aerial,591,33,200x200,0.1--1,RGB
-`Inria Aerial Image Labeling`_,S,Aerial,360,,"5,000x5,000",0.3,RGB
+`Inria Aerial Image Labeling`_,S,Aerial,360,2,"5,000x5,000",0.3,RGB
```
Comment (adamjstewart) [inline docs/api/non_geo_datasets.csv:15]: Binary semantic segmentation so 2 classes

File: docs/api/non_geo_datasets.csv [hunk 8]
```diff
@@ -1,34 +1,34 @@
...
 `COWC`_,"C, R","CSUAV AFRL, ISPRS, LINZ, AGRC","388,435",2,256x256,0.15,RGB
 `Kenya Crop Type`_,S,Sentinel-2,"4,688",7,"3,035x2,016",10,MSI
-`Deep Globe Land Cover Challenge`_,S,DigitalGlobe +Vivid,803,7,"2,448x2,448",0.5,RGB
-`DFC2022`_,S,Aerial,,15,"2,000x2,000",0.5,RGB
+`Deep Globe Land Cover`_,S,DigitalGlobe +Vivid,803,7,"2,448x2,448",0.5,RGB
+`DFC2022`_,S,Aerial,"3,981",15,"2,000x2,000",0.5,RGB
 `ETCI2021 Flood Detection`_,S,Sentinel-1,"66,810",2,256x256,5--20,SAR
 `EuroSAT`_,C,Sentinel-2,"27,000",10,64x64,10,MSI
 `FAIR1M`_,OD,Gaofen/Google Earth,"15,000",37,"1,024x1,024",0.3--0.8,RGB
 `Forest Damage`_,OD,Drone imagery,"1,543",4,"1,500x1,500",,RGB
```
Comment (adamjstewart) [inline docs/api/non_geo_datasets.csv:12]: Still don't know the resolution of this. It looks like Dan Morris (AI for Earth) is associated with this project and may know. There is also contact info on the dataset webpage: https://lila.science/datasets/forest-damages-larch-casebearer/

File: docs/api/geo_datasets.csv (hunk 2 above)
Comment (calebrob6) [reply docs/api/geo_datasets.csv:4]: "generated" to me is model output vs. manual interpretation

File: docs/api/non_geo_datasets.csv (hunk 8 above)
Comment (calebrob6) [reply docs/api/non_geo_datasets.csv:12]: Sent an email to the author

Comment (calebrob6) [reply docs/api/non_geo_datasets.csv:12]: FYI Dan is at google now

File: docs/api/geo_datasets.csv (hunk 3 above)
Comment (calebrob6) [reply docs/api/geo_datasets.csv:5]: Would go with "Masks"

**Review by calebrob6 — CHANGES_REQUESTED** [review]
Can you change "mask" --> "masks"? I'll open an issue to update resolution on Forest Damages dataset.

**Review by calebrob6 — APPROVED** [review]

**Review by calebrob6 — APPROVED** [review]

**Pull request merged**